### PR TITLE
Add README for MariaDB Client

### PR DESCRIPTION
**Pull request by alexandrevicenzi**
This is related to #993.

### CODE REVIEW

File: src/bci_build/package/mariadb-client/README.md.j2 [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# {{ image.pretty_name }} container image
+
+## Description
+
+MariaDB is an open-source, multi-threaded, relational database management system. It's a backward-compatible branch of the MySQL Community Server that provides a drop-in replacement for MySQL.
+
+This image contains the [MariaDB client](https://mariadb.com/kb/en/mariadb-command-line-client/) and utilities.
```
Comment (alexandrevicenzi) [inline src/bci_build/package/mariadb-client/README.md.j2:7]: Should I list all utilities included in the image? Or can we assume that users will know what comes in this container?

I had no clue. First I looked at what was in `/usr/bin` and then I looked at what binaries were installed by mariadb-client package.

File: src/bci_build/package/mariadb-client/README.md.j2 [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# {{ image.pretty_name }} container image
+
+## Description
```
Comment (dcermak) [inline src/bci_build/package/mariadb-client/README.md.j2:3]:
```suggestion

{% include 'badges.j2' %}

## Description
```

File: src/bci_build/package/mariadb-client/README.md.j2 (hunk 1 above)
Comment (dcermak) [reply src/bci_build/package/mariadb-client/README.md.j2:7]: I think it makes sense to list the shipped client binaries

Comment (alexandrevicenzi) [reply src/bci_build/package/mariadb-client/README.md.j2:7]: The issue is that the binaries start with `mariadb` and `mysql`, not sure if I list both, or just MariaDB version.

This is why I felt leaving it out would be better.

Comment (dcermak) [reply src/bci_build/package/mariadb-client/README.md.j2:7]: The `mysql*` variants are only symlinks to the respective `mariadb*` binary. Hence it's imho sufficient to mention the `mariadb*` versions

File: src/bci_build/package/mariadb-client/README.md.j2 (hunk 2 above)
Comment (alexandrevicenzi) [reply src/bci_build/package/mariadb-client/README.md.j2:5]: Added.

File: src/bci_build/package/mariadb-client/README.md.j2 (hunk 1 above)
Comment (alexandrevicenzi) [reply src/bci_build/package/mariadb-client/README.md.j2:9]: I added mariadb only in a new section.

**Review by dirkmueller — APPROVED** [review]

File: src/bci_build/package/mariadb-client/README.md.j2 [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Usage
+
+To connect to a MariaDB instance, run the following command:
+
+```ShellSession
+$ podman run -it --rm {{ image.pretty_reference }} mariadb -h $HOST_IP -u root -p
+```
+
+To dump all databases, run the following command:
```
Comment (dmpop) [inline src/bci_build/package/mariadb-client/README.md.j2:19]:
```suggestion
Use the command below to dump all databases:
```

File: src/bci_build/package/mariadb-client/README.md.j2 [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- mariadb-admin
+- mariadb-check
+- mariadb-dump
+- mariadb-import
+- mariadb-show
+- mariadbd-safe-helper
+
+## Backup and restore
+
+Information on how to perform backup and restore can be found on the MariaDB Knowledge Base [Container Backup and Restoration](https://mariadb.com/kb/en/container-backup-and-restoration/).
```
Comment (dmpop) [inline src/bci_build/package/mariadb-client/README.md.j2:39]:
```suggestion
The MariaDB Knowledge Base [Container Backup and Restoration](https://mariadb.com/kb/en/container-backup-and-restoration/) provides further information on how to perform backup and restore.
```

**Review by dirkmueller — APPROVED** [review]

**Pull request merged**